code/code/disc/disc_offense.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __DISC_OFFENSE_H
```

Comment:
`#pragma once` is supported on all compliers we might conceivably care about

Reply:
Seems to work.